### PR TITLE
Releasing patch 1.0.2

### PR DESCRIPTION
**Pull request by MartinSchobben**
I intend to submit this patch to CRAN. This contains the updated documentation of Mariana. However, due to the missing dependency in the the irods demo, all GitHub workflows fail for now (irods/irods_demo#46). Any idea how to circumvent this for now?

### CODE REVIEW

File: tests/testthat/add-data-collections/list-88a93f.R
Comment (MartinSchobben) [reply tests/testthat/add-data-collections/list-88a93f.R:3]: Yes, I have to still make new http snapshots. Those are the conflicts. But the problems relate more to the irods demo to create those snapshots.

Comment (trel) [reply tests/testthat/add-data-collections/list-88a93f.R:3]: PR https://github.com/irods/irods_demo/pull/47 is merged

**Comment by MartinSchobben** [comment]
Submitted patch to CRAN. Let's wait if that goes through and then merge this PR with a tag for 0.1.2.

**Comment by MartinSchobben** [comment]
It's on CRAN now: https://cran.r-project.org/web/packages/rirods/index.html

**Review by trel — APPROVED** [review]
seems good.

**Comment by MartinSchobben** [comment]
Can you merge this and release with tag 0.1.2?

**Comment by trel** [comment]
lots of conflicting files at the moment - not yet mergeable.

**Comment by trel** [comment]
Oh, I see that now.   And my name is on all of them.   Why is anything getting pushed weekly if there are no code changes?   That feels surprising and weird/wrong.

**Comment by MartinSchobben** [comment]
I added this to enable testing without an iRODS server/internet connection (e.g. on the CRAN servers). I have the weekly updates to capture potential changes in the HTTP response. But maybe weekly is a bit much.

**Comment by trel** [comment]
Oh, I see.   Is there a way to remove/ignore the timestamps from responses files?    Then you could detect any 'real' changes more easily?

**Comment by MartinSchobben** [comment]
ah yes, that should be possible as well. I entails changing `inst/httptest2/redact.R`.

**Comment by MartinSchobben** [comment]
I fixed the HTTP snapshots that now not include dates anymore. The tag for version 0.1.2 now belongs to commit 4144b54. This commit is the same as the "truth", from the point of view CRAN (R).

**Comment by trel** [comment]
so this is okay to be merged, as is?  five commits?

**Comment by MartinSchobben** [comment]
yes, that is fine. Up to commit https://github.com/irods/irods_client_library_rirods/commit/4144b545566069a01c8a54870f1051532775024e is needed to stay in sync with CRAN (0.1.2) the last two fix the snapshots in the dev version. 

Regarding fixing the automated snapshot generation. I now have to update the snapshots again :-)

**Comment by trel** [comment]
Got it.

One last note - when rebasing, the SHA of that commit will change (again).

If that matters, then we need to make sure we get things merged (with a stable SHA) before we declare a release.
If not, then we should be good to go.

Merging!

**Comment by MartinSchobben** [comment]
That makes sense. Never thought about that. In the normal situation the R developer will be push a package to CRAN from the main branch. That is not possible in our situation, where I push to CRAN from my own development branch. This is, however, all only considered as good practice and not mandatory.

**Comment by trel** [comment]
You can still push to CRAN from main - just have to wait until we merge first?

**Comment by MartinSchobben** [comment]
Right! That would be better then.

**Pull request merged**
